spec/models/factories_spec.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.factories.map(&:name).each do |factory_name|
```

Comment:
I'm not sure what FactoryGirl.lint does but this just called .valid? on each of the factories.

http://robots.thoughtbot.com/testing-your-factories-first